src/Pages/OATEditorPage/Internal/OATErrorPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { getEditorPageStyles } from '../OATEditorPage.Styles';
+
+const OATErrorPage = ({ error }) => {
+    const { t } = useTranslation();
+    const EditorPageStyles = getEditorPageStyles();
```

Comment:
Should this be `editorPageStyles`? I.e. lower case `E`. Depends on the pattern that's used elsewhere.